metadata-ingestion-modules/gx-plugin/setup.py

```diff
@@ -0,0 +1,163 @@
+import os
+import pathlib
+
+import setuptools
+
+package_metadata: dict = {}
+with open("./src/datahub_gx_plugin/__init__.py") as fp:
+    exec(fp.read(), package_metadata)
+
+
+def get_long_description():
+    root = os.path.dirname(__file__)
+    return pathlib.Path(os.path.join(root, "README.md")).read_text()
+
+
+rest_common = {"requests", "requests_file"}
+
+# TODO: Is this required here??
+pydantic_no_v2 = {
+    # pydantic 2 makes major, backwards-incompatible changes - https://github.com/pydantic/pydantic/issues/4887
+    # Tags sources that require the pydantic v2 API.
+    "pydantic<2",
+}
+# TODO: Can we move away from sqllineage and use sqlglot ??
+sqllineage_lib = {
+    "sqllineage==1.3.8",
+    # We don't have a direct dependency on sqlparse but it is a dependency of sqllineage.
+    # There have previously been issues from not pinning sqlparse, so it's best to pin it.
+    # Related: https://github.com/reata/sqllineage/issues/361 and https://github.com/reata/sqllineage/pull/360
+    "sqlparse==0.4.4",
+}
+
+_version: str = package_metadata["__version__"]
+_self_pin = (
+    f"=={_version}" if not (_version.endswith("dev0") or "docker" in _version) else ""
+)
```

Comment:
seems like we're not using the self_pin anywhere?

Reply:
right. Should we be using this ? Its used in airflow-plugin. Not used in dagster-plugin with this comment - `# Ignoring the dependency below because it causes issues with the vercel built wheel install`

Reply:
Fixed it by updating _self_pin to check for `dev1` along with `dev0`